base/exports.jl

```diff
@@ -950,6 +950,7 @@ export
     sizeof,
 
 # tasks and conditions
+    channeled_tasks,
```

Comment:
Note that lowercase `channel` will be a new export. Will need a new function as we have to return a tuple of channel and task. This need prevents us from adding this functionality to either channel/task constructors, or even adding a keyword option to `schedule`